src/binder/bind/bind_ddl.cpp

```diff
@@ -24,40 +22,94 @@ namespace binder {
     const parser::Statement& statement) {
-    auto& createRelClause = (CreateRelClause&)statement;
-    auto tableName = createRelClause.getTableName();
+    auto& createTableClause = (CreateTableClause&)statement;
+    auto tableName = createTableClause.getTableName();
     if (catalog.getReadOnlyVersion()->containTable(tableName)) {
         throw BinderException("Rel " + tableName + " already exists.");
     }
-    auto boundProperties = bindProperties(createRelClause.getPropertyNameDataTypes());
+    auto createInfo = createTableClause.getInfo();
+    auto boundProperties = bindProperties(createInfo->propertyNameDataTypes);
     for (auto& boundProperty : boundProperties) {
         if (boundProperty->getDataType()->getLogicalTypeID() == LogicalTypeID::SERIAL) {
             throw BinderException("Serial property is not supported in rel table.");
         }
     }
-    auto relMultiplicity = getRelMultiplicityFromString(createRelClause.getRelMultiplicity());
-    return make_unique<BoundCreateRelClause>(tableName, std::move(boundProperties), relMultiplicity,
-        bindNodeTableID(createRelClause.getSrcTableName()),
-        bindNodeTableID(createRelClause.getDstTableName()));
+    auto extraInfo = (RelExtraCreateTableInfo*)createInfo->extraInfo.get();
+    auto srcTableID = bindNodeTableID(extraInfo->srcTableName);
+    auto srcTableSchema =
+        (NodeTableSchema*)catalog.getReadOnlyVersion()->getTableSchema(srcTableID);
+    auto srcPkDataType = srcTableSchema->getPrimaryKey()->getDataType();
+    auto dstTableID = bindNodeTableID(extraInfo->dstTableName);
+    auto dstTableSchema =
+        (NodeTableSchema*)catalog.getReadOnlyVersion()->getTableSchema(dstTableID);
+    auto dstPkDataType = dstTableSchema->getPrimaryKey()->getDataType();
+    auto relMultiplicity = getRelMultiplicityFromString(extraInfo->relMultiplicity);
+    auto boundExtraInfo = std::make_unique<BoundRelExtraCreateTableInfo>(
+        relMultiplicity, srcTableID, dstTableID, srcPkDataType->copy(), dstPkDataType->copy());
+    auto boundCreateInfo = std::make_unique<BoundCreateTableInfo>(
+        tableName, std::move(boundProperties), std::move(boundExtraInfo));
+    return make_unique<BoundCreateTable>(
+        StatementType::CREATE_REL_TABLE, tableName, std::move(boundCreateInfo));
+}
+
+static constexpr char RDF_IRI[] = "_IRI";
+static constexpr char RDF_PREDICT_ID[] = "_PREDICT_ID";
+static constexpr char RDF_NODE_TABLE_SUFFIX[] = "_RESOURCE";
+static constexpr char RDF_REL_TABLE_SUFFIX[] = "_TRIPLES";
+
+std::unique_ptr<BoundStatement> Binder::bindCreateRdfGraphClause(
+    const parser::Statement& statement) {
+    auto& createTableClause = (CreateTableClause&)statement;
+    auto rdfGraphName = createTableClause.getTableName();
+    auto stringType = std::make_unique<LogicalType>(LogicalTypeID::STRING);
+    // RDF node (resource) table
+    auto nodeTableName = rdfGraphName + RDF_NODE_TABLE_SUFFIX;
```

Comment:
I think we can refactor these to a function:
1. getRdfGraphNodeTableName()
2. getRdfGraphRelTableName()
Which internally appends the suffix to the rdfGraphName